packages/css/src/plugins/__tests__/sort-shorthand-declarations.test.ts

```diff
@@ -20,6 +20,125 @@ describe('sort shorthand vs. longhand declarations', () => {
     process.env.BROWSERSLIST = 'last 1 version';
   });
 
+  it('sorts against multi-property classes: simple', () => {
+    // We need to make sure this `{-webkit-text-decoration-color:initial;text-decoration-color:initial}` doesn't "block" sorting
+    const actual = transform(outdent`
+      ._zulph461{gap:3px}
+      ._bfhk1ayf{background-color:red}
+      ._1jmq18uv{-webkit-text-decoration-color:initial;text-decoration-color:initial}
```

Comment:
ah is this from autoprefixer? I guess the 1 property per class name assumption is incorrect after all

sorry for introducing the bug 😔

Reply:
Nah, 99% sure it was there before because I just `return 0`'d arrays (and this is an array of properties)